lib/active_merchant/billing/gateways/elavon.rb

```diff
@@ -379,7 +379,9 @@ def build_xml_request
       end
 
       def commit(request)
-        request = "xmldata=#{request}".delete('&')
+        request = request.force_encoding('iso-8859-1')
+        request = "xmldata=#{request.force_encoding('utf-8')}".delete('&')
```

Comment:
Not super relevant to your point, but we should be able to get rid of this `.delete('&')` if the encoding tweaks work.